gitbook-docs/connection.md

```diff
@@ -1,63 +1,97 @@
 # Discovery and Connection Establishment
 
-#### Service Discovery
+## Service Discovery
 
-A Signal K server SHOULD advertise its services over mDNS/Bonjour. The server MUST use the service types
+A Signal K server SHOULD advertise its services using [DNS Service Discovery
+(DNS-SD)](https://tools.ietf.org/html/rfc6763) via Mutlicast DNS (mDNS); also known as Bonjour. The server MUST provide
+DNS [Service (SRV) Records](https://en.wikipedia.org/wiki/SRV_record) and [Text (TXT)
+Records](https://en.wikipedia.org/wiki/TXT_record) describing the Signal K interfaces it provides. These service
+identifiers are:
 
-* `_signalk-http._tcp` for http API
-* `_signalk-ws._tcp` for WebSocket
-* `_signalk-https._tcp` for HTTPS API
-* `_signalk-wss._tcp` for secure WebSocket
+* `_http._tcp` and/or `_https._tcp` for the server's web interface
```

Comment:
http://www.dns-sd.org/serviceTypes.html

>Rather than having an entirely different DNS-SD service type for https, we recommend using the same redirection mechanism: advertise a plain "http" service, which consists of nothing except an HTTP redirection to the desired "https" URL.